examples/mode3draw/mode3draw.zig

```diff
@@ -1,44 +1,44 @@
-const GBA = @import("gba").GBA;
-const Color = @import("gba").Color;
-const Mode3 = @import("gba").Mode3;
-const LCD = @import("gba").LCD;
+const gba = @import("gba");
+const Color = gba.Color;
+const Mode3 = gba.Bitmap(Color, 240, 160);
```

Comment:
`Mode3 = gba.Bitmap(gba.Color, 240, 160)` should be in the `gba` module as good defaults.